platforms/hyperledger-fabric/configuration/roles/create/ca-tools/orderer/tasks/main.yaml

```diff
@@ -115,25 +115,81 @@
   when: network.env.proxy == 'ambassador'
 
 ############################################################################################
-# This task copies the msp folder from the respective CA Tools CLI
-# to the Ansible container
-- name: Copy the msp folder from the ca tools
+# This task copies the msp admincerts from vault
+- name: Copy the msp admincerts from vault
   shell: |
-    export CA_TOOL_CLI=$(KUBECONFIG={{ kubernetes.config_file }}  kubectl get po -n {{ component_name }} | grep "ca-tools" | awk '{print $1}')
-    KUBECONFIG={{ kubernetes.config_file }} kubectl cp {{ component_name }}/${CA_TOOL_CLI}:crypto-config/{{ component_type }}Organizations/{{ component_name }}/msp ./build/crypto-config/{{ component_type }}Organizations/{{ component_name }}/msp
+    vault kv get -field=admincerts {{ vault.secret_path | default('secretsv2') }}/crypto/ordererOrganizations/{{ component_name }}/users/admin/msp > Admin@{{ component_name }}-cert.pem
+    mkdir -p ./build/crypto-config/ordererOrganizations/{{ component_name }}/msp/admincerts/
+    mv Admin@{{ component_name }}-cert.pem ./build/crypto-config/ordererOrganizations/{{ component_name }}/msp/admincerts/
+  environment:
+    VAULT_ADDR: "{{ vault.url }}"
+    VAULT_TOKEN: "{{ vault.root_token }}"
+  
+############################################################################################
+# This task copies the msp cacerts from vault
+- name: Copy the msp cacerts from vault
+  shell: |
+    vault kv get -field=cacerts {{ vault.secret_path | default('secretsv2') }}/crypto/ordererOrganizations/{{ component_name }}/users/admin/msp > ca-{{ component_name }}-{{ item.external_url_suffix }}-8443.pem
+    mkdir -p ./build/crypto-config/ordererOrganizations/{{ component_name }}/msp/cacerts/
+    mv ca-{{ component_name }}-{{ item.external_url_suffix }}-8443.pem ./build/crypto-config/ordererOrganizations/{{ component_name }}/msp/cacerts/
+  environment:
+    VAULT_ADDR: "{{ vault.url }}"
+    VAULT_TOKEN: "{{ vault.root_token }}"
+  when: network.env.proxy != 'none'  
+
+############################################################################################
+# This task copies the msp tlscacerts from vault
+- name: Copy the msp tlscacerts from vault
```

Comment:
Please rename the task and the above comment to `Fetch the msp tlscacerts from vault`